src/callstack.cr

```diff
@@ -367,7 +367,7 @@ struct CallStack
       @@base_address : UInt64|UInt32|Nil
 
       protected def self.read_dwarf_sections
-        Debug::ELF.open(PROGRAM_NAME) do |elf|
+        Debug::ELF.open(Process.executable_path || PROGRAM_NAME) do |elf|
```

Comment:
```suggestion
        return unless path = Process.executable_path
        Debug::ELF.open(path) do |elf|
```

Reply:
Done!

Reply:
What if `exception_path` is present but `File.open` fails?

Reply:
Then we're back in the same position, but I think making that more robust should be done in another PR - it'll have to be done for the other executable types as well.